### PR TITLE
URL does not need to be a base url

### PR DESCRIPTION
**Pull request by jzoldak**
Also add tests for the repo. Now 15% covered. :|
@benpatterson

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+sudo: false
+language: python
+branches:
+    only:
+      - master
+python:
+  - "2.7"
+install:
+  - pip install -r test-requirements.txt
+  - pip install coveralls
```
Comment (benpatterson) [inline .travis.yml]: It's a nit, but we could install coveralls as part of the after_success.

File: edx_sitespeed/edx_sitespeed.py [hunk 2]
```diff
@@ -35,9 +38,12 @@ def login(email, password, base_url, auth_user=None, auth_pass=None):
     else:
         auth = None
 
+    base_url = get_base_url(page_url)
```
Comment (benpatterson) [inline edx_sitespeed/edx_sitespeed.py]: sorry if i'm being base...but I don't see where `page_url` is declared?

Comment (jzoldak) [inline edx_sitespeed/edx_sitespeed.py]: Nope, that's a typo. Good catch. Should be "url"

File: edx_sitespeed/edx_sitespeed.py [hunk 3]
```diff
@@ -48,37 +54,36 @@ def login(email, password, base_url, auth_user=None, auth_pass=None):
...
+                        default=None)
+    parser.add_argument('--auth_pass', help='basic auth password',
+                        default=None)
     args = parser.parse_args()
 
     session_info = login(
-        args.email, args.password, args.base_url, auth_user=args.auth_user, auth_pass=args.auth_pass)
-    create_headers_file(session_info['session_key'], session_info['session_id'], session_info['csrf_token'])
+        args.email, args.password, args.url, auth_user=args.auth_user,
+        auth_pass=args.auth_pass)
```
Comment (benpatterson) [inline edx_sitespeed/edx_sitespeed.py]: Down the road, will we need an API token?

**Comment by benpatterson** [comment]
:+1:

File: edx_sitespeed/edx_sitespeed.py (hunk 3 above)
Comment (jzoldak) [inline edx_sitespeed/edx_sitespeed.py]: Yeah, this way assumes that the django application is up and running, and the API authenticates users through the django session middleware.
Could be expanded later to use OAUTH2 instead if we need to.

**Pull request merged**